docs/source/bird-rr-config.rst

```diff
@@ -0,0 +1,149 @@
+Configuring BIRD as a BGP Route Reflector
+=========================================
+
+These instructions will take you through installing BIRD as a BGP route
+reflector, and updating your other BIRD instances to speak to your new route
+reflector.
```

Comment:
Since a few reactions to Calico have been along the lines of 'but BGP is big and scary and hard to configure', is it worth including a link here to some kind of BGP/Route Reflection 101 guide?